### PR TITLE
brill: init at 4.000.073

### PR DESCRIPTION
**Pull request by trespaul**
## Description of changes

Add the [Brill font](https://brill.com/page/BrillFont/brill-typeface).

It has a custom license, under which it is free for non-commercial use. I saw other licenses that are free for non-commercial use labelled as `unfree`, so I chose the generic `unfree` license. Not sure if this is correct.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by katexochen — CHANGES_REQUESTED** [review]
Hey, thanks for your contribution. Currently,  your package isn't referenced anywhere and thus won't be build. Please add it in `pkgs/top-level/all-packages.nix` or, even better, to `pkgs/by-name`. You can find additional explanation in the [Readme](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md) of that directory.

**Comment by trespaul** [comment]
I was poking around after submitting this PR and realised I might have had to do that, so thanks for the info; I'll update soon.

**Review by katexochen — CHANGES_REQUESTED** [review]
Result of `nixpkgs-review pr 267038` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>brill</li>
  </ul>
</details>

File: pkgs/data/fonts/brill/default.nix
[the remark this answers is not in the document]
Comment (trespaul) [reply pkgs/data/fonts/brill/default.nix:1]: I think something must have gone wrong when I squashed the commits. I'll remove the old one in a moment.

File: pkgs/by-name/br/brill/package.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
```
[the remark this answers is not in the document]
Comment (trespaul) [reply pkgs/by-name/br/brill/package.nix]: Done! Thanks!

File: pkgs/by-name/br/brill/package.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    hash = "sha256-ugmEIkeBzD/4C9wkVfbctEtnzI8Kw+YD6KGcbk4BAf4=";
+    stripRoot = false;
+  };
+
+  installPhase = with finalAttrs; ''
+    runHook preInstall
+
+    install -Dm644 *.ttf          -t $out/share/fonts/truetype
+    install -Dm644 *agreement.pdf -t $out/share/licenses/${pname}-${version}
+    install -Dm644 *use.pdf       -t $out/share/doc/${pname}-${version}
```
Comment (katexochen) [inline pkgs/by-name/br/brill/package.nix:18]:
```suggestion
    install -Dm644 *agreement.pdf -t $out/share/licenses/${pname}
    install -Dm644 *use.pdf       -t $out/share/doc/${pname}
```
`version` is already part of the store path.

**Review by katexochen — COMMENTED** [review]
otherwise lgtm :)

**Comment by katexochen** [comment]
Good, now we just need to clean up the commit history. The first two commits should stay.

**Comment by trespaul** [comment]
Thanks for all your help.

**Review by katexochen — APPROVED** [review]
Thanks for your contribution. :)

**Comment by emilytrau** [comment]
Thanks and welcome to nixpkgs! :)

**Pull request merged**